articles/azure/azs-how-create-vm-image-portal.md

```diff
@@ -0,0 +1,117 @@
+---
+title: How to create a custom image from a VM on Azure Stack using the UKCloud Azure Stack portal | UKCloud Ltd
+description: Provides details on how to create a custom image from a VM using the portal
+services: azure-stack
+author: Daniel Brennand
+reviewer:
+lastreviewed:
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Create a custom image from a VM - Portal
+toc_fullpath: Users/How To/azs-how-create-vm-image-portal.md
+toc_mdlink: azs-how-create-vm-image-portal.md
+---
+
+# How to create a custom image from a VM on Azure Stack using the UKCloud Azure Stack portal
+
+## Overview
+
+You can create an image resource from a generalised virtual machine (VM) that is stored as either a managed disk or an unmanaged disk in a storage account. You can then use the image to create multiple VMs.
+
+This article explains how to create a custom image from a VM on Azure Stack, which you can then use to deploy other VMs.
+
+## Prerequisites
+
+To complete the steps in this article, you must have appropriate access to a subscription in the Azure Stack portal.
+
+## [1. Generalise your VM](#tab/tabid-1)
+
+> [!WARNING]
+> Once you've generalised a VM, you **cannot** log back into it.
+
+## [2. Creating the image](#tab/tabid-2)
+
+> [!WARNING]
+> Capturing a VM image will make it unusable and cannot be undone.
+
+1. Log in to the [Azure Stack portal](https://portal.frn00006.azure.ukcloud.com/).
```

Comment:
Can we change this step to be more like how we do it elsewhere in the docs? e.g. https://docs.ukcloud.com/articles/azure/azs-how-create-vm-portal.html#creating-a-virtual-machine

Reply:
Not really sure what was different. I just added `For more detailed instructions, see [Getting Started Guide for UKCloud for Microsoft Azure](azs-gs.md).`